compose-app/src/main/java/com/mapbox/maps/testapp/compose/MainActivity.kt

```diff
@@ -0,0 +1,131 @@
+package com.mapbox.maps.testapp.compose
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.animation.core.LinearOutSlowInEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import com.mapbox.geojson.Point
+import com.mapbox.maps.*
+import com.mapbox.maps.extension.compose.MapboxMap
+import com.mapbox.maps.extension.compose.annotation.CircleAnnotation
+import com.mapbox.maps.extension.style.style
+import com.mapbox.maps.plugin.gestures.generated.GesturesSettings
+
+private const val LATITUDE = 60.239
+private const val LONGITUDE = 25.004
+
+public class MainActivity : ComponentActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContent {
+      HomeScreen()
+    }
+  }
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun HomeScreen() {
+  var mapGesturesState by remember {
+    mutableStateOf(GesturesSettings())
+  }
+  var mapStyleState by remember {
+    mutableStateOf(style(Style.MAPBOX_STREETS) {})
+  }
+  var annotationPoint by remember {
+    mutableStateOf(Point.fromLngLat(LONGITUDE, LATITUDE))
+  }
+  var annotationSize by remember {
+    mutableStateOf(10.0)
+  }
+
+  var annotationPoint1 by remember {
+    mutableStateOf(Point.fromLngLat(LONGITUDE + 0.01, LATITUDE))
+  }
+  var annotationSize1 by remember {
+    mutableStateOf(5.0)
+  }
+  var zoom by remember {
+    mutableStateOf(12.0f)
+  }
+  val zoomLevel: Float by animateFloatAsState(
+    zoom,
+    animationSpec = tween(
+      durationMillis = 1000,
+      easing = LinearOutSlowInEasing
+    )
+  )
+  Box(Modifier.fillMaxSize()) {
```

Comment:
could you please test it with some height? I can't set correct center position with height = 200.dp

Reply:
I tried with Modifier.height(200.dp).fillMaxWidth() and it works correctly.